ui/src/components/WorkersTable/index.jsx

```diff
@@ -152,112 +249,158 @@ export default class WorkersTable extends Component {
+                  tooltipTitle={latestTask.run.started}
+                  textToCopy={latestTask.run.started}
+                  text={<DateDistance from={latestTask.run.started} />}
+                />
               ) : (
-                <em>n/a</em>
+                <TableCell>n/a</TableCell>
               )}
-            </TableCell>
-            {latestTask?.run ? (
-              <CopyToClipboardTableCell
-                tooltipTitle={latestTask.run.started}
-                textToCopy={latestTask.run.started}
-                text={<DateDistance from={latestTask.run.started} />}
-              />
-            ) : (
-              <TableCell>n/a</TableCell>
-            )}
-            {latestTask?.run?.resolved ? (
-              <CopyToClipboardTableCell
-                tooltipTitle={latestTask.run.resolved}
-                textToCopy={latestTask.run.resolved}
-                text={<DateDistance from={latestTask.run.resolved} />}
-              />
-            ) : (
-              <TableCell>n/a</TableCell>
-            )}
-            <TableCell>
-              {quarantineUntil ? (
-                formatDistanceStrict(new Date(), parseISO(quarantineUntil), {
-                  unit: 'day',
-                })
+              {latestTask?.run?.resolved ? (
+                <CopyToClipboardTableCell
+                  tooltipTitle={latestTask.run.resolved}
+                  textToCopy={latestTask.run.resolved}
+                  text={<DateDistance from={latestTask.run.resolved} />}
+                />
               ) : (
-                <em>n/a</em>
+                <TableCell>n/a</TableCell>
               )}
-            </TableCell>
-          </TableRow>
-        )}
-        headers={[
-          'Worker Group',
-          'Worker ID',
-          'Last Active',
-          'First Claim',
-          'Most Recent Task',
-          'Task State',
-          'Task Started',
-          'Task Resolved',
-          'Quarantined',
-        ]}
-        {...props}
-      />
+              <TableCell>
+                {quarantineUntil ? (
+                  formatDistanceStrict(new Date(), parseISO(quarantineUntil), {
+                    unit: 'day',
+                  })
+                ) : (
+                  <em>n/a</em>
+                )}
+              </TableCell>
+              <TableCell>
+                {providerId !== NULL_PROVIDER ? (
+                  <IconButton
+                    title="Terminate Worker"
+                    className={classes.button}
+                    name={`${taskQueueId}`}
+                    onClick={this.handleDialogActionOpen(
+                      taskQueueId,
+                      workerGroup,
+                      workerId
+                    )}>
+                    <DeleteIcon size={iconSize} />
+                  </IconButton>
+                ) : (
+                  <Label mini status="warning" className={classes.button}>
+                    Scheduled for termination
```

Comment:
why is it scheduled for termination?

Reply:
Same https://github.com/taskcluster/taskcluster/pull/5448#discussion_r881864891